lib/jekyll-feed/meta-tag.rb

```diff
@@ -5,10 +5,29 @@ class MetaTag < Liquid::Tag
     # Use Jekyll's native relative_url filter
     include Jekyll::Filters::URLFilters
 
+    def initialize(tag_name, args, tokens)
+      super
+      @args = args.strip
+    end
+
     def render(context)
       @context = context
-      attrs    = attributes.map { |k, v| %(#{k}="#{v}") }.join(" ")
-      "<link #{attrs} />"
+      @collection = nil
+      @category   = nil
+
+      if @args.strip == "include: all"
```

Comment:
There's no need to `strip` again. It's been handled in the `initialize` method.
```suggestion
      if @args == "include: all"
```